src/properties.jl

```diff
@@ -304,16 +307,17 @@ end
 
 coco_init(phi::EuclideanMatrix{CT}, l, m, μ, T, A) where {CT<:Real} = (
       (l == 2 && abs(m) <= 2 && abs(μ) <= 2)
-         ? vec([EuclideanMatrix{CT}(conj.(transpose(mrmatrices[(m,μ,i,j)]))) for i=1:3 for j=1:3])
+         ? vec([EuclideanMatrix(conj.(transpose(mrmatrices[(m,μ,i,j)]))) for i=1:3 for j=1:3])
          : coco_zeros(phi, l, m, μ, T, A)  )
 
 
-coco_init(::EuclideanMatrix{CT}) where {CT<:Real} = [EuclideanMatrix{CT}(SMatrix{3,3,Float64,9}([1.0,0,0,0,1.0,0,0,0,1.0]))]       
```

Comment:
I think you want 
```
EuclideanMatrix{CT}(SMatrix{3,3,CT}([1.0,0,0,0,1.0,0,0,0,1.0]))
```
If I remember correctly this is the implementation for zero-correlations. So you are right that identity is the only zero-correlation there is, but you probably need to specify the right eltype.

Reply:
included in issue #100